src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -1,14 +1,34 @@
 package core.basesyntax.service;
 
+import core.basesyntax.RegistrationException;
 import core.basesyntax.dao.StorageDao;
 import core.basesyntax.dao.StorageDaoImpl;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int MINIMUM_AGE = 18;
+    private static final int MINIMUM_PASSWORD = 6;
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
-        return null;
+        if (user == null || user.getLogin() == null
+                || user.getPassword() == null || user.getAge() == null) {
+            throw new RegistrationException("Invalid data. User can't be null");
```

Comment:
in Common mistakes is said not to make combo of if, maybe it's better to separate them